Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/ConfigParser.kt

```diff
@@ -14,17 +16,24 @@ class ConfigParser @Inject constructor(
     private val cwaConfigMapper: CWAConfig.Mapper,
     private val keyDownloadConfigMapper: KeyDownloadConfig.Mapper,
     private val exposureDetectionConfigMapper: ExposureDetectionConfig.Mapper,
-    private val riskCalculationConfigMapper: RiskCalculationConfig.Mapper
+    private val riskCalculationConfigMapper: RiskCalculationConfig.Mapper,
+    private val exposureWindowRiskCalculationConfigMapper: ExposureWindowRiskCalculationConfig.Mapper
 ) {
 
     fun parse(configBytes: ByteArray): ConfigMapping = try {
+        // TODO replace with actual v2 config
```

Comment:
Not sure how far Matthias is with the default app config... it that's ready, you could already use the new v2 default app config.

Reply:
v2 migration is currently a seperate ticket tracked in EXPOSUREAPP-3414 when this is merged we can remove the conflicting changes from this branch